pages/autocomplete/index.js

```diff
@@ -195,7 +195,7 @@ const AutoCompleteDemo = () => {
 
                     <h5>Multiple</h5>
                     <span className="p-fluid">
-                        <AutoComplete value={selectedCountries} suggestions={filteredCountries} completeMethod={searchCountry} field="name" multiple onChange={(e) => setSelectedCountries(e.value)} aria-label="Countries" />
+                        <AutoComplete value={selectedCountries} suggestions={filteredCountries} completeMethod={searchCountry} field="name" multiple onChange={(e) => setSelectedCountries(e.value)} aria-label="Countries" selectionLimit={3} />
```

Comment:
I would remove this here but add it up in the Section titled "Multiple Mode" where the description and example of Multiple Mode is..

![image](https://user-images.githubusercontent.com/4399574/199963616-dbb6d129-0831-4106-a64f-676f264329cc.png)

Reply:
Thanks for the suggestion, will do!

Reply:
I added a description and example below the existing one, let me know if I should just combine the sentences and examples instead or if the wording is off. Thanks.